### ISSUE TITLE
[Feature Request] Change arrows by scroll bar

### ISSUE THREAD

**Issue by Amnesia1000** [citra-emu/citra#4622]
All options that are handled with percentages (3D, speed limit, play coins) should have a scroll bar to handle it, as this is easier to use and faster to set, instead of arrows to raise / lower percentage.
It is a very visual request, but it would give a greater uniformity to the emulator, since some options use the arrows and others the scroll.
Also you could change all to arrows, although personally I prefer the scroll. The main idea as I said is to make the emulator more uniform and handle all options equally.

**Comment by FearlessTobi** [comment]
The play coins and the speed limit have no maximum, so a scroll bar makes absolutely no sense for them. 
I'm not opposed to a scroll bar for the 3d slider though.

**Comment by Amnesia1000** [comment]
Play coins mas is 300...

**Comment by Amnesia1000** [comment]
I disagree, in maximum speed are generally used integer values (50, 100, 200, 300 max.), more than 300% is equal to disable the max speed and play coins the max is 300, and not who does not always put them in 300 ... also by default is set at 43 and no one has complained ...
But as I mentioned the idea is to give the emulator uniformity, if the scroll do not like, change everything to dates.

Another option is not to use arrows or scroll and leave an editable text box only where one can put the value manually.
